bin/README.md

```diff
@@ -1,10 +0,0 @@
-# bin directory
-
-This directory is placed into `PATH` environment variable so that tools located
-here are available to all tests (this is done in
-[`run_ocsci.py`](../run_ocsci.py) wrapper).
-
-The directory is intentionally empty (there should be no other files committed
-there). Command line tools like `oc` and `openshift-install` are installed
-there during deployment (see
-[test_ocs_basic_install.py](../tests/ecosystem/deployment/test_ocs_basic_install.py)).
```

Comment:
Why is this file deleted? I consider having a README file in every important directory useful to keep it's purpose visible, which helps us with maintenance and usage in the long term.

Reply:
I have no idea,
I will restore it in next PR.
Sorry

Reply:
I added a response here, but can't see it anymore, so sorry if I'm replying twice. I would suggest to just return the file in a separate commit in this pull request, so that when this PR is merged via github's Squash and Merge, the change won't be visible as if nothing happened to the readme file in the final commit.

Reply:
I have this file in my local repo and it doesn't seem to be visible here.
In case it will not be part of the merge, I will add it right after the merge

Reply:
It is not going to be part of the merge in it's current state, as the file is removed in commit f92cbaeab758283b2c983d717f991cac7f80cdfa.

If you have the file in your local repository, you don't see it in `git status` output because `bin` directory is in gitignore. I was thinking if there is anything wrong with our code, but didn't noted anything which could have caused this.

Anyway, you can fix it quickly via:

```
$ rm bin/README.md # just in case
$ git checkout c9fc4fd30d8eba7cab3d5787de0eeb6a679a534b -- bin/README.md
$ git commit -m 'reverting README.md removal'
```

Then, when this is merged via "Squash and Merge", the resulting commit won't touch this file as if it were never deleted.